CHANGES.rst

```diff
@@ -10,7 +10,10 @@ New Features
 
 - ``astropy.convolution``
 
-  - Major change in convolution behavior and keyword arguments (#5782)
+  - Major change in convolution behavior and keyword arguments.   Additional
+    details are in the API section. (#5782)
```

Comment:
I think all of those got fixed when we started to use the release script. Anyway, this was more like a heads up, I need to do changelog fixes anyway during releasing, so can fix this, too.